app/queries/create_school_link_mutator.rb

```diff
@@ -47,7 +47,7 @@ def create_school_link
       end
 
     params[:kind] = kind
-
+    params[:sort_index] = SchoolLink.where(kind: kind).count
```

Comment:
Migrate to the new format for mutations

Reply:
`SchoolLink.where(kind: kind).count` this will return the count of links in the school.
```rb
SchoolLink.where(kind: kind, school: resource_school).maximum(:sort_index).to_i + 1
```